sections/Chapter 7 Sprint 5/Review.tex

```diff
@@ -1 +1,23 @@
 \section{Review}
+This section will review the accomplishment for the 5th sprint as well as changes in the \knox{} environment. 
+
+The goal for this sprint was to implement functionality to support the user stories defined in section \ref{Userstories}. 
+
+\subsubsection*{What has been accomplished?}
+The user stories which refers to the RDF database has been implemented but are all yet to be deployed to the \knox{} pipeline. This means the acceptance criteria proposed in section \ref{acceptCriteriaSprint5} are fulfilled but since the features was not deployed to the pipeline the primary goal for the sprint has not been accomplished. 
```

Comment:
```suggestion
The user stories which refers to the RDF database has been implemented but are all yet to be deployed to the \knox{} pipeline. This means the acceptance criteria proposed in section \ref{acceptCriteriaSprint5} are fulfilled but since the features were not deployed to the pipeline the primary goal for the sprint has not been accomplished. 
```